web/styles/app_components.css

```diff
@@ -160,12 +160,8 @@ input::placeholder {
                font-size and one with 16px base font-size. We multiply
                by (16 / 14 * 0.8 = 0.9143) so rem equivalent value
                will remain the same in the legacy 14px mode, while
-               scaling the size up for 16px mode. The font-size will
-               resolve in pixels.
-               TODO: Refactor this part to either use `em` or `rem`
-               and not this temporary hack of using calc() and
-               variables. */
-        font-size: calc(0.9143 * var(--base-font-size-px));
+               scaling the size up for 16px mode. */
+        font-size: 0.9143em;
```

Comment:
I'm not sure why we didn't do this previously ... so am not sure if we're ready to do this.

We probably also don't need most of that comment if it's now `em`.

Reply:
The 0.9143 value, as the comment states, was in deference to the old 16px-based rem value (or 12.8px, for 0.8rem). So yes, I believe I was over-thinking this at the time and it would've been safe to move directly to ems at the time. 

Still, a couple of spot checks on elements taking `.small` would be useful to document this change, especially at smaller font sizes, to make sure that nothing is disturbed by this. The only way I could see this being a problem is if there's an occurence of `.small` inside an inherited `font-size` value that's other than the base font-size value.

Reply:
Yeah; I'm not sure I'm aware of regions that we intend to be that way. There's kinda a lot to audit. 

`$ git grep small[^a-z] web/templates/ | wc -l` = 62.

I'm not aware of places we make entire containers not the default font size, except popovers, and `git grep small[^a-z] web/templates/ | grep popover` is empty, so probably this is fine. I'll adjust the comment.

Reply:
Updated comment is just this:
`+        /* 16 / 14 * 0.8 = 0.9143, from original 0.8rem */`